torch/_inductor/ir.py

```diff
@@ -2588,20 +2650,21 @@ def get_unbacked_symbol_uses(self):
 
     def make_loader(self):
         # Inline constants and index_expressions
-        can_inline = (
+        if (
             hasattr(self.data, "make_loader")
```

Comment:
This check is broken now since all IRNodes have a `make_loader` attr.

Reply:
oops. Surprised the tests didn't catch this...

I'll grep for more such `hasattr` calls.